cmd/docker-app/split.go

```diff
@@ -1,30 +1,46 @@
 package main
 
 import (
-	"github.com/docker/app/internal"
+	"os"
+	"strings"
+
 	"github.com/docker/app/internal/packager"
 	"github.com/docker/cli/cli"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
 var splitOutputDir string
 
 func splitCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "split [<app-name>] [-o output_dir]",
+		Use:   "split [<app-name>] [-o output]",
 		Short: "Split a single-file application into multiple files",
 		Args:  cli.RequiresMaxArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			appname, cleanup, err := packager.Extract(firstOrEmpty(args))
+			extractedApp, err := packager.ExtractWithOrigin(firstOrEmpty(args))
 			if err != nil {
 				return err
 			}
-			defer cleanup()
-			return packager.Split(appname, splitOutputDir)
+			defer extractedApp.Cleanup()
+			inPlace := splitOutputDir == ""
```

Comment:
Shouldn't that be 
```
inPlace := splitOutputDir == "" || splitOutputDir == extractedApp.OriginalAppName
```
?

Reply:
No I'd rather not. That command will fail and not overwrite if you pass any other existing directory.